packages/flutter/lib/src/material/text_selection_material.dart

```diff
@@ -0,0 +1,191 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
```

Comment:
No need for the `_material` suffix.  The file is in the `material` directory.  We have lots of duplicated file names in different directories.